src/main/java/com/reckue/post/repositories/CommentRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.reckue.post.repositories;
+
+import com.reckue.post.models.Comment;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
Please, add javadoc before class.

Reply:
I think it would be better to write that  interface is responsible for connecting to MongoDB, because the CRUD operations we use in services and thier realizations.